.github/workflows/after-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+name: After deployments
+
+on: [deployment_status]
+
+jobs:
+  fetch-vercel-logs:
+    name: 'Vercel Build Log'
+    if: github.event.deployment_status.state != 'pending'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v2
+      - name: Install Vercel CLI
+        run: npm i --silent --no-progress vercel
+      - name: CLICK HERE TO SEE THE VERCEL BUILD LOG
+        run: FORCE_COLOR=true vercel logs -n 10000 --scope commercetools --token="${{ secrets.VERCEL_TOKEN }}" ${{ github.event.deployment_status.target_url }}
```

Comment:
This is not needed here

Reply:
I would like to ease contributions without having to make everybody a vercel user, also in the kit.  Just one small pice in being more open for contributions.  But no emotions attached.

Reply:
I see, but I would reconsider this if we get requests/feedback from potential contributors.

For now we all have access to the logs, so I don't see a need to have this job in the docs-kit.